### PR TITLE
Create newsletter for Feb-March 2022

### PR DESCRIPTION
**Pull request by malvikasharan**
Adding newsletter for Feb-Mar 2022

### CODE REVIEW

**Review by EstherPlomp — APPROVED** [review]

File: communications/newsletters/newsletter_32_Mar2022.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+![Malvika's tweet inviting application to this Book Dash.](images/2022-03-bookdash-03.jpg)
+_[Tweet by Malvika](https://twitter.com/MalvikaSharan/status/1505906141921951745?s=20&t=xHT7RiowG8hRJytNqMpPjQ) sharing report from [the 6th Book Dash in 2022](https://hackmd.io/@turingway/bookdash-nov2021)_
+
+#### Express Your Interest to Join the Planning Committee or Host a Local Meet Up
+
+Previous Book Dash attendees are welcome to send their application to participate again in the Book Dash. They are particularly encouraged to indicate their interest to join the **Book Dash Review and Planning Committee**. A Book Dash committee members are facilitators and mentors who support other attendees’ contributions in one or multiple of the contribution sessions based on their interest and availability. We are also looking to support **small scale local meetups** if multiple members from the same city are participating. In the application form, please indicate your interest if you would like to attend or host if a local meet-up in your city will be possible.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply communications/newsletters/newsletter_32_Mar2022.md]: Added in the file.

File: communications/newsletters/newsletter_32_Mar2022.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+These roles are maternity covers for Kirstie Whitaker, TPS Programme Director and TPS Theme Lead, who will be available for handover and knowledge transfer conversations until May 2022.
+
+#### TPS Research Project Manager
+
+A Research Project Manager will coordinate and support work across the TPS programme and They will play a central role in assisting with the overall delivery process. This position will also be working closely with _The Turing Way_ core team members. They will be supported to share their expertise in project design and collaborative skills by creating new content, reviewing existing chapters, or promoting the project to new audiences. See this [Twitter thread by Arielle Bennett](https://twitter.com/biotechchat/status/1500810442217857024?s=20&t=ZhCl7r3zrp8d1NstXaNFHA) for details and [apply online](https://cezanneondemand.intervieweb.it/turing/jobs/research_project_manager_tools_practices_and_systems_21254/en/).
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply communications/newsletters/newsletter_32_Mar2022.md]: Integrated the change.

**Comment by malvikasharan** [comment]
Thank you so much @EstherPlomp and @GeorgiaHCA for the review. 🌻

**Comment by malvikasharan** [comment]
Newsletter is sent! 🌮

**Pull request merged**